srcpkgs/hexer/template

```diff
@@ -11,7 +11,7 @@ maintainer="Aloz1 <kno0001@gmail.com>"
 license="BSD"
```

Comment:
please use spdx identifier

Reply:
```
Copyright (c) 1995,1996 Sascha Demetrio
All rights reserved.

Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions
are met:
1. Redistributions of source code must retain the above copyright
   notice, this list of conditions and the following disclaimer.
   If you modify any part of HEXER and redistribute it, you must add
   a notice to the `README' file and the modified source files containing
   information about the  changes you made.  I do not want to take
   credit or be blamed for your modifications.
2. Redistributions in binary form must reproduce the above copyright
   notice, this list of conditions and the following disclaimer in the
   documentation and/or other materials provided with the distribution.
   If you modify any part of HEXER and redistribute it in binary form,
   you must supply a `README' file containing information about the
   changes you made.
3. The name of the developer may not be used to endorse or promote
   products derived from this software without specific prior written
   permission.

HEXER WAS DEVELOPED BY SASCHA DEMETRIO.
THIS SOFTWARE SHOULD NOT BE CONSIDERED TO BE A COMMERCIAL PRODUCT.
THE DEVELOPER URGES THAT USERS WHO REQUIRE A COMMERCIAL PRODUCT
NOT MAKE USE OF THIS WORK.

DISCLAIMER:
THIS SOFTWARE IS PROVIDED BY THE DEVELOPER ``AS IS'' AND
ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
ARE DISCLAIMED.  IN NO EVENT SHALL THE DEVELOPER BE LIABLE
FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS
OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
SUCH DAMAGE.
```
It is mostly `BSD-3-Clause` with a added blame part. Should I just add it as `BSD-3-Clause` ?